cobalt/android/apk/app/src/main/java/dev/cobalt/coat/CobaltActivity.java

```diff
@@ -350,6 +363,54 @@ protected void onCreate(Bundle savedInstanceState) {
     videoSurfaceView = new VideoSurfaceView(this);
     addContentView(
         videoSurfaceView, new LayoutParams(LayoutParams.MATCH_PARENT, LayoutParams.MATCH_PARENT));
+
+    initializeJavaBridge();
+  }
+
+  /**
+   * Initializes the Java Bridge to allow communication between Java and JavaScript.
+   * This method injects Java objects into the WebView and loads corresponding JavaScript code.
+   */
+  private void initializeJavaBridge() {
+
+    WebContents webContents = getActiveWebContents();
+    if (webContents == null) {
+        // WebContents not initialized yet, post a delayed runnable to check again
+        new Handler(Looper.getMainLooper()).postDelayed(new Runnable() {
+            @Override
+            public void run() {
+                initializeJavaBridge(); // Recursive call to check again
+            }
+        }, JAVA_BRIDGE_INITIALIZATION_DELAY_MILLI_SECONDS);
+        return;
+    }
+
+    // --- Initialize the Java Bridge ---
+
+    // 1. Gather all Java objects that need to be exposed to JavaScript.
+    javaScriptAndroidObjectList.add(new AmatiDeviceInspector(this));
+
+    // 2. Use JavascriptInjector to inject Java objects into the WebView.
```

Comment:
Is this WebView or WebContents?

Reply:
JavascriptInjector.fromWebContents, updated the comment